lua/modules/AnimatedGlitchedScans.lua

```diff
@@ -0,0 +1,60 @@
+----------------------------------------------------------------------------------------------------
+-- Module Initialization
+----------------------------------------------------------------------------------------------------
+
+function Init()
+	local m = NewWebsiteModule()
+	m.ID                       = -- TODO --
```

Comment:
@Slasar41, not sure how to add `ID`, but I was able to manually add this lua to my system and was able to view managa and download chapters.

Reply:
It's GUID, there's a plugin called `ToolBucket` in Notepad++ that can generate GUID.